CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# CHANGELOG
```

Comment:
We can retroactively add this manually, or start doing the auto changelog updates for the next release

Reply:
Actually, I forgot this was in docs/changelog.md, so I moved that here.